### PR TITLE
Annonymous platform identifiers for Device, User, Logon Session

### PR DESCRIPTION
**Pull request by radujipa**
Using hashed (SHA256) Windows security identifiers (SIDs).

### CODE REVIEW

File: KpcUwpCore/PlatformDetection/PlatformIdentifiers.cs [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            // The SID structure that identifies the user that is currently associated
+            // with the specified object. If no user is associated with the object,
+            // the value returned in the buffer pointed to by lpnLengthNeeded is zero.
+            // Note that SID is a variable length structure. You will usually make a
+            // call to GetUserObjectInformation to determine the length of the SID
+            // before retrieving its value.
+            private const int UOI_USER_SID = 4;
+
+            // Retrieves the thread identifier of the calling thread.
+            [DllImport("kernel32.dll")]
```
Comment (tombettany) [inline KpcUwpCore/PlatformDetection/PlatformIdentifiers.cs:72]: I'm guessing that UWP/Windows Store doesn't have a problem with loading DLLs that are already present?

**Review by tombettany — APPROVED** [review]
A bit difficult to actually write a useful unit test here but otherwise looking good.

File: KpcUwpCore/PlatformDetection/PlatformIdentifiers.cs (hunk 1 above)
Comment (radujipa) [reply KpcUwpCore/PlatformDetection/PlatformIdentifiers.cs:72]: When validating the POC for this I did make a full release of it in the Sandbox app without issues so I'd expect it to work 🤞

**Comment by radujipa** [comment]
I did scratch my head a bit how to even write one test for it, a lot of it is dependent on env setup...

**Pull request merged**